pkg/controller/restore-operator/sync.go

```diff
@@ -28,6 +24,7 @@ const (
 	//
 	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
 	maxRetries = 15
+	tmpdir     = "/tmp"
```

Comment:
What's this var for?
Isn't it duplicate?

Reply:
yeah, it is duplicated from backup-operator.

Reply:
I mean duplicate in restore-operator/backup_handler.go too